src/OSPSuite.Core/Services/PredictedVsObservedChartService.cs

```diff
@@ -38,6 +38,12 @@ public interface IPredictedVsObservedChartService
       ///    <paramref name="observationColumns" /> and adjusts the axes names
       /// </summary>
       void ConfigureAxesDimensionAndTitle(IReadOnlyList<DataColumn> observationColumns, PredictedVsObservedChart chart);
+
+      /// <summary>
+      ///    Creates new deviation line with fold value <paramref name="foldValue" />
+      /// </summary>
+      IEnumerable<DataRepository> AddDeviationLine(float foldValue, List<DataColumn> observationColumns, PredictedVsObservedChart chart,
```

Comment:
not clear why you return an enumeration of DataREpository here. 
CAn you explain what this is?